File: app/src/main/java/com/pluu/webtoon/common/Const.java
package com.pluu.webtoon.common;

/**
 * Created by nohhs on 2015-03-02.
 */
public class Const {
	public static final String LOG_TAG = "PluuToon";
	public static final String EXTRA_API = "EXTRA_API";
	public static final String EXTRA_POS = "EXTRA_POS";
	public static final String EXTRA_EPISODE = "EXTRA_EPISODE";
	public static final String EXTRA_MAIN_COLOR = "EXTRA_MAIN_COLOR";
	public static final String EXTRA_STATUS_COLOR = "EXTRA_STATUS_COLOR";

	private static final String RATE_FORMAT = "평점 : %.2f";

	public static final String MAIN_FRAG_TAG = "main_frag_tag";

	public static String getRateNameByRate(String data) {
		return String.format(RATE_FORMAT, Double.valueOf(data));
	}

}

File: app/src/main/java/com/pluu/webtoon/common/Const.kt
package com.pluu.webtoon.common

/**
 * Application Const Set
 * Created by pluu on 2017-04-29.
 */
object Const {
    val LOG_TAG = "PluuToon"
    const val EXTRA_API = "EXTRA_API"
    const val EXTRA_POS = "EXTRA_POS"
    const val EXTRA_EPISODE = "EXTRA_EPISODE"
    const val EXTRA_MAIN_COLOR = "EXTRA_MAIN_COLOR"
    const val EXTRA_STATUS_COLOR = "EXTRA_STATUS_COLOR"

    const val CONFIG_NAME = "service"

    private val RATE_FORMAT = "평점 : %.2f"

    const val MAIN_FRAG_TAG = "main_frag_tag"
    const val DETAIL_FRAG_TAG = "detail_frag_tag"

    fun getRateNameByRate(data: String) = RATE_FORMAT.format(data.toDouble())

}
